web/app/landing.module.css

```
@@ -3887,3 +3892,283 @@
+  opacity: 0.85;
+  margin: 4px 0;
+}
+
+.howConnector svg {
+  width: 40px;
+  height: 64px;
+}
+
+.howConnectorPath {
+  fill: none;
+  stroke: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)